bridge/bridge/api/static/openapi/openapi.yaml

```diff
@@ -0,0 +1,919 @@
+          description: The provided address is invalid for filtering wrap requests.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+        500:
+          description: Unexpected internal error.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+
+  /wrap/requests/hash/{transaction_hash}:
+    get:
+      summary: Returns wrap requests by request or payout transaction hash.
+      description: Retrieves wrap requests matched by either the request hash or associated payout hash.
+      tags:
+        - Wrap
+      operationId: getWrapRequestsByHash
+      parameters:
+        - $ref: '#/components/parameters/TransactionHash'
+        - $ref: '#/components/parameters/Offset'
+        - $ref: '#/components/parameters/Limit'
+        - $ref: '#/components/parameters/Sort'
+        - $ref: '#/components/parameters/PayoutStatus'
+      responses:
+        200:
+          description: Wrap requests corresponding to the specified transaction hash.
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/WrapRequest'
+        400:
+          description: Invalid transaction hash format.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+        500:
+          description: Unexpected internal error.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+
+  /wrap/errors:
+    get:
+      summary: Returns wrap errors.
+      description: Retrieves a paginated list of errors encountered while processing wrap requests.
+      tags:
+        - Wrap
+      operationId: getWrapErrors
+      parameters:
+        - $ref: '#/components/parameters/Offset'
+        - $ref: '#/components/parameters/Limit'
+        - $ref: '#/components/parameters/Sort'
+      responses:
+        200:
+          description: List of wrap errors stored by the bridge.
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/WrapError'
+        400:
+          description: Invalid request parameters.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+        500:
+          description: Unexpected internal error.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+
+  /wrap/errors/{address}:
+    get:
+      summary: Returns wrap errors for a specific sender address.
+      description: Retrieves wrap errors linked to the specified request sender address.
```

Comment:
Same comments as above.

The similar `/wrap/requests/{address}` endpoint can filter by both sender and receiver addresses, but this one only filters by sender. This is a bit surprising so it deserves a comment. Or better yet: make them behave the same way, so it's easier to understand and to explain :D